webrtc.html

```diff
@@ -12160,7 +12160,8 @@ <h3>
           The {{RTCIceTransportState}} of an {{RTCIceTransport}} may change
           because a candidate pair with a usable connection was found and
           selected or it may change without the selected candidate pair
-          changing. The selected pair and {{RTCIceTransportState}} are related
+          changing. The <dfn class="export" data-lt="change of selected candidate pair or state">change of the selected pair and
+          {{RTCIceTransportState}}</dfn> are related
```

Comment:
I think this needs to go in the paragraph below where the algorithm is invoked. E.g.

          When the [= ICE Agent =] indicates that an {{RTCIceTransport}} has
          changed either the selected candidate pair, the
          {{RTCIceTransportState}} or both, the user agent MUST queue a task
          that runs these [steps to change pair and state] algorithm:

Reply:
Done